pkgs/development/tools/java/visualvm/default.nix

```diff
@@ -1,27 +1,26 @@
-{ stdenv, fetchzip, lib, makeWrapper, jdk, gtk2, gawk }:
+{ stdenv, fetchzip, lib, makeWrapper, makeDesktopItem, jdk, gtk2, gawk }:
 
 stdenv.mkDerivation rec {
-  name = "visualvm-1.3.9";
+  name = "visualvm-1.4";
 
   src = fetchzip {
-    url = "https://github.com/visualvm/visualvm.src/releases/download/1.3.9/visualvm_139.zip";
-    sha256 = "1gkdkxssh51jczhgv680i42jjrlia1vbpcqhxvf45xcq9xj95bm5";
+    url = "https://github.com/visualvm/visualvm.src/releases/download/1.4/visualvm_14.zip";
```

Comment:
Not a blocker but you could attempt to re-use the `version` attribute here to simplify bumps. It seems you could simply remove the dots in the version string for the archive basename.

Reply:
Ok I will try it out.

Reply:
You know I just realized that I am only changing the url. How do I remove the sources from the nix store to ensure the url actually works when I run nix-build?

Reply:
The quick way is to just perturb the hash while you're trying things out ... or maybe use `nix-prefetch-url -A ...` which should ensure the fetcher actually runs. Anyway, this is not all that important, so I'd be happy to integrate this as-is,. the version string reuse can be iterated upon later.